heuristics/heuristic_2.py

```diff
@@ -0,0 +1,120 @@
+from numpy import ndarray
+from typing import Tuple, Dict
+
+from lib.util import distance_nb_coups
+from lib.positions import get_human_positions, get_our_positions, get_opponent_positions
+
+
+def human_impact(
+    our_positions: Dict[Tuple[int, int], int],
+    humans_positions: Dict[Tuple[int, int], int],
+    opponent_positions: Dict[Tuple[int, int], int],
+    human_win_weight: int,
+):
+    human_impact_score = 0
+    for human_position, human_size in humans_positions.items():
+        for our_position, our_size in our_positions.items():
+            distance = distance_nb_coups(our_position, human_position)
+            if human_size <= our_size:
+                human_impact_score += (
+                    human_win_weight
+                    * (our_size + human_size)
+                    / ((distance ** 2) * our_size)
```

Comment:
Curious as to why multiplying by `our_size` 🤔

Reply:
It was a relative weight but actually it was changed in the new version